doc/src/manual/getting-started.md

```diff
@@ -104,4 +104,39 @@ command-line-options).
 
 A curated list of useful learning resources to help new users get started can be found on the [learning](https://julialang.org/learning/) page of the main Julia web site.
 
+REPL can itself be used as a learning resource by switching into the help mode, you can switch to help mode by pressing `?`. Typing a keyword in help mode will fetch the documentation for the same along with examples.
+
+```
+help?> begin
+search: begin disable_sigint reenable_sigint
+
+  begin
+
+  begin...end denotes a block of code.
+
+  begin
+      println("Hello, ")
+      println("World!")
+  end
+
+  Usually begin will not be necessary, since keywords such as function and let
+  implicitly begin blocks of code. See also ;.
+
+  begin may also be used when indexing to represent the first index of a
+  collection or the first index of a dimension of an array.
+
+  Examples
+  ≡≡≡≡≡≡≡≡≡≡
+
+  julia> A = [1 2; 3 4]
+  2×2 Array{Int64,2}:
+   1  2
+   3  4
+
+  julia> A[begin, :]
+  2-element Array{Int64,1}:
+   1
+   2
```

Comment:
I don't think the whole text is essential, just enough to show the principle of it
```suggestion
```